### PR TITLE
LFA-1927 adding get max extension limit from config and adding to page render

### PR DESCRIPTION
**Pull request by SVarallo**

### CODE REVIEW

File: src/test/controllers/extension.limit.reached.spec.unit.ts [hunk 1]
```diff
@@ -20,6 +20,7 @@ const ERROR_MSG = "Sorry, there is a problem with the service";
 const ERROR_TITLE = "Sorry, there is a problem with the service - GOV.UK";
 const AFTER_TWELVE_MONTHS_MSG = "We cannot grant an extension to the filing deadline";
 const AFTER_TWELVE_MONTHS_TITLE = "You cannot use this service";
+const MAX_EXTENSION_PERIOD_IN_MONTHS = 12;
```
Comment (mattch1) [inline src/test/controllers/extension.limit.reached.spec.unit.ts:23]: If this is an expected value I'd recommend including "expected" in the variable name.

**Comment by mattch1** [comment]
Are you going to update the company details controller? - there is a TODO in there for this ticket.

**Comment by SVarallo** [comment]
> Are you going to update the company details controller? - there is a TODO in there for this ticket.

I'll add that to the PR. Wasn't aware that was part of the task

File: src/test/controllers/extension.limit.reached.spec.unit.ts (hunk 1 above)
Comment (SVarallo) [reply src/test/controllers/extension.limit.reached.spec.unit.ts]: Updated the variable name 👍

**Pull request merged**